<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGoalUI</class>
 <widget class="QDialog" name="NewGoalUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NewGoalUI</string>
  </property>
  <widget class="QScrollArea" name="AddStep_ScrollingArea">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>751</width>
     <height>111</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>749</width>
      <height>109</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>248</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>2 step</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="StepTitle_LineEdit_2"/>
      </item>
      <item>
       <widget class="QSpinBox" name="AddStep_TimEdit_2"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>50</y>
       <width>248</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>4 step</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="StepTitle_LineEdit_4"/>
      </item>
      <item>
       <widget class="QSpinBox" name="AddStep_TimEdit_4"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>248</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>3 step</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="StepTitle_LineEdit_3"/>
      </item>
      <item>
       <widget class="QSpinBox" name="AddStep_TimEdit_3"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>248</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>5 step</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="StepTitle_LineEdit_5"/>
      </item>
      <item>
       <widget class="QSpinBox" name="AddStep_TimEdit_5"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>248</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>1 step</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="StepTitle_LineEdit_1"/>
      </item>
      <item>
       <widget class="QSpinBox" name="AddStep_TimEdit_1"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>19</y>
     <width>751</width>
     <height>257</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="NewGoal_Label">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Create your own goal</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Title_Line">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Your title</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="Description_TextEdit">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>500</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Save_pusButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Category_ComboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
